package.json

```diff
@@ -81,7 +81,7 @@
     "fetch-ponyfill": "^7.1.0",
     "i18next": "22.4.12",
     "idb": "^7.1.1",
-    "inputmask": "^5.0.8",
+    "inputmask": "5.0.8",
```

Comment:
Upgrading to 5.0.9 was causing the tests to fail, so I've pinned it here and asked for a ticket to be created to investigate/upgrade the dependency